File: chapter1/permutation.py
# Check to see if one string is permutation of other string

def permutation(str1, str2):
    newStr1 =  ''.join(sorted(str1))
    newStr2 =  ''.join(sorted(str2))

    if newStr1 == newStr2:
        return True
    else:
        return False

print(permutation('god', 'dog'))

print(permutation('cat', 'dog'))

File: chapter1/permutation.js
function permutation(str1, str2) {
  newStr1 = str1.split("").sort().join("");
  newStr2 = str1.split("").sort().join("");

  if (newStr1 == newStr2) {
    return true;
  }

  return false;
}

console.log(permutation("dog", "god"));

module.exports = permutation;
